engine/src/conversion/api.rs

```diff
@@ -221,6 +221,7 @@ pub(crate) enum SpecialMemberKind {
     MoveConstructor,
     Destructor,
     AssignmentOperator,
+    GeneratedAccessor,
```

Comment:
Unfortunately I don't think you can add this here. I didn't add this bit of code, but AIUI "special member" has a specific meaning in C++ - https://cplusplus.com/doc/tutorial/classes2/ - and a generated accessor isn't one of them.

Reply:
Oops, of course. I was using this to identify which are the accessors that we generated during function analysis, specifically for giving nicer rust names. I'll need to come up with a different strategy. Do you think it would make sense to have the accessor generation add that info in the analysis, similar to how POD types are labelled?